src/OpenAL/OpenToolkit.OpenAL/AudioContext.cs

```diff
@@ -228,7 +223,7 @@ internal bool IsCurrent
         /// <summary>
```

Comment:
Line 97-100, I cant select for some reason..... 

```c
public AudioContext()
    : this(null)
{
}
```

can this not be shortened to

```c
public AudioContext(AudioDevice audioDevice = null)
{
...whatever happens in that ctor called by this(null)
}
```

Reply:
I'm not sure this file is going to survive this PR so I don't think we should worry too much about it

Reply:
Ok that wasn't obvious to me.

Reply:
Yeah I just haven't gotten to it yet :)

Reply:
This file is no more!